### PR TITLE
Adding buildscript for go-control-plane v0.9.0 & v0.9.7

### PR DESCRIPTION
**Pull request by bahetiamit**
## Checklist

- [X] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [X] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: g/go-control-plane/go-control-plane_v0.9.0_RHEL_8.4.sh [hunk 1]
```diff
@@ -0,0 +1,130 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : github.com/envoyproxy/go-control-plane
+# Version       : v0.9.0
+# Source repo   : https://github.com/go-stack/stack.git
```
Comment (seth-priya) [inline g/go-control-plane/go-control-plane_v0.9.0_RHEL_8.4.sh:5]: is this pointing to the correct source code repo.?

File: g/go-control-plane/go-control-plane_v0.9.0_RHEL_8.4.sh [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+# Tested on     : RHEL 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Amit Baheti (aramswar@in.ibm.com)
+#
```
Comment (seth-priya) [inline g/go-control-plane/go-control-plane_v0.9.0_RHEL_8.4.sh:9]: We have recently introduced additional fields in the script header, can you please refer to the latest templates and / or guidelines doc and include those? Mainly "Language", "Travis-Check" etc.

File: g/go-control-plane/go-control-plane_v0.9.0_RHEL_8.4.sh [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+#               package and/or distribution. In such case, please
+#               contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=github.com/envoyproxy/go-control-plane
+PACKAGE_VERSION=v0.9.0
+PACKAGE_PATH=https://github.com/envoyproxy/go-control-plane.git
+
+yum -y update && yum install -y nodejs nodejs-devel nodejs-packaging npm python38 python38-devel ncurses git jq wget gcc-c++
```
Comment (seth-priya) [inline g/go-control-plane/go-control-plane_v0.9.0_RHEL_8.4.sh:22]: do we need to install all these dependencies for this particular package?

File: g/go-control-plane/go-control-plane_v0.9.7_RHEL_8.4.sh [hunk 4]
```diff
@@ -0,0 +1,130 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : github.com/envoyproxy/go-control-plane
+# Version       : v0.9.7
+# Source repo   : https://github.com/go-stack/stack.git
+# Tested on     : RHEL 8.4
```
Comment (seth-priya) [inline g/go-control-plane/go-control-plane_v0.9.7_RHEL_8.4.sh:6]: do we really need separate build scripts for both versions?

File: g/go-control-plane/go-control-plane_v0.9.0_RHEL_8.4.sh [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+export GOPATH=/root/go
+
+mkdir -p output
+
+OS_NAME=$(cat /etc/os-release | grep ^PRETTY_NAME | cut -d= -f2)
+
+export PATH=$GOPATH/bin:$PATH
+export GO111MODULE=on
+
+function test_with_master_without_flag_u(){
```
Comment (seth-priya) [inline g/go-control-plane/go-control-plane_v0.9.0_RHEL_8.4.sh:39]: it's okay to simplify the template to use only specific set of instructions needed for building/test the particular package

**Comment by bahetiamit** [comment]
@seth-priya Updated the script to follow new guidelines

**Review by seth-priya — APPROVED** [review]
LGTM

**Pull request merged**